boards/FSM/Src/App/states/App_AllStates.c

```diff
@@ -40,19 +37,11 @@ void App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     float right_torque_req = App_CanMsgs_dcm_invr_command_message_torque_command_invr_decode(
         App_CanRx_DCM_INVR_COMMAND_MESSAGE_GetSignal_TORQUE_COMMAND_INVR(can_rx));
     float fsm_torque_limit = App_CanTx_GetPeriodicSignal_FSM_TORQUE_LIMIT(can_tx);
-    if (left_torque_req > fsm_torque_limit || right_torque_req > fsm_torque_limit)
-    {
-        error_count++;
-    }
-    else
-    {
-        error_count = 0;
-    }
+    if (left_torque_req > fsm_torque_limit || right_torque_req > fsm_torque_limit) error_count++;
```

Comment:
IMO this redundancy check is completely pointless. It only trips if the DCM is going crazy. In such a case we have BSPD and DCM Watchdog working anyway. Thoughts from anyone? @gtaharaedmonds @pasitlaothamatas @kirons96